kubernetes/helm_charts/secor/config/secor.common.properties

```diff
@@ -23,12 +23,12 @@ secor.kafka.topic_blacklist=
 
 # Choose what to fill according to the service you are using
 # in the choice option you can fill S3, GS, Swift or Azure
-cloud.service=Azure
+cloud.service={{ $.Values.storage_type }}
 
 # AWS authentication credentials.
 # Leave empty if using IAM role-based authentication with s3a filesystem.
-aws.access.key=
-aws.secret.key=
+aws.access.key={{ $.Values.s3_access_key }}
```

Comment:
@ddevadat  The secor.common.properties should be left as-is and we should add secor.s3.properties to accommodate s3-like object stores. Please refer to secor.azure.properties file and also secor.partition.properties file where secor.azure.properties is imported.

Reply:
its restructured now